examples/fabric-admin/commands/common/IcdManager.cpp

```diff
@@ -32,6 +34,20 @@ void IcdManager::OnCheckInComplete(const chip::app::ICDClientInfo & clientInfo)
     {
         mDelegate->OnCheckInCompleted(clientInfo);
     }
+
```

Comment:
I think all of this code needs to be in the delegate not in the IcdManager, we need to be giving a `activeDurationMs` the value we got, not just this hard-coded 30000.